src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -63,10 +64,10 @@ function MoneyRequestConfirmPage(props) {
     const {isOffline} = useNetwork();
     const {windowWidth} = useWindowDimensions();
     const prevMoneyRequestId = useRef(props.iou.id);
-    const iouType = useRef(lodashGet(props.route, 'params.iouType', ''));
-    const isDistanceRequest = MoneyRequestUtils.isDistanceRequest(iouType.current, props.selectedTab);
+    const iouType = useInitialValue(() => lodashGet(props.route, 'params.iouType', ''));
```

Comment:
What's the actual performance benefit here? Or is this just for consistency?

Reply:
As you wrote - there is negligible performance benefit, so it's more about:
1. Consistency.
2. Clearer intent - now is easier to read this as 'I will take initial value on first render, and I won't change it'